lib/bosh/workspace/project_deployment.rb

```diff
@@ -28,7 +28,15 @@ def merged_file
     end
 
     def manifest
-      @manifest ||= Psych.load(ERB.new(File.read(file)).result)
+      return @manifest unless @manifest.nil?
+      renderer = Bosh::Template::Renderer.new(context: stub.to_json)
+      @manifest = Psych.load(renderer.render(file))
+    end
+
+    def stub
+      return @stub unless @stub.nil?
+      stub_file = File.expand_path(File.join(file_dirname, "../stubs", file_basename))
+      @stub = File.exist?(stub_file) ? Psych.load(File.read(stub_file)) : {}
```

Comment:
Sure. But it was easier for me to add tests for `#manifest` (because of mocks) method that involved cases with and without stubs. Take a look [here](https://github.com/Altoros/bosh-workspace/blob/add-user-stubs-for-deployments/spec/project_deployment_spec.rb#L19).